app/assets/javascripts/share.js

```diff
@@ -4,67 +4,89 @@ $(document).on('ready page:load', function() {
 });
 
 function pageLoad() {
-    //datatable
-    (function() {
-        if ($('.data_table')) {
-            $('.data_table').dataTable({
-                paging: true,
-                dom: 'Rlfrtip',
-                "order": [
-                    [0, "desc"]
-                ],
-                "aoColumnDefs": [{
-                    "bSortable": false,
-                    "aTargets": [-1, -2]
-                }]
-            });
-        }
-    })();
+  var but = $( "#but" );
+  var save = $( "#save" );
+  var editor = $( "#editor" );
+  var wisywyg_container = $( "#wisywyg_container" );
 
-    // fileinput
-    if ($('.file_input')) {
-        $(".file_input").fileinput({
-            maxFilesNum: 1
-        });
-    }
+  //datatable
+  (function() {
+      if ($('.data_table')) {
```

Comment:
This condition probably might not work as expected. As well as all other like this: `if($('...')) { ...`